templates/actors/group/inventory.hbs

```diff
@@ -0,0 +1,44 @@
+<section class="tab {{ tab.cssClass }}" data-tab="{{ tab.id }}" data-group="{{ tab.group }}">
+
+    <section class="sidebar">
+        {{#each members}}
+        <div class="encumbrance card" data-uuid="{{ uuid }}">
+            {{#with encumbrance}}
+            <div class="meter progress" role="meter" aria-valuemin="0" aria-valuenow="{{ pct }}"
+                 aria-valuetext="{{ value }}" aria-valuemax="{{ max }}" style="--bar-percentage: {{ pct }}%">
+                <div class="label">
+                    <i class="fa-solid fa-weight-hanging" inert></i>
+                    <span class="value">{{ value }}</span>
+                    <span class="separator">&sol;</span>
+                    <span class="max">{{ max }}</span>
+                </div>
+            </div>
+            {{/with}}
```

Comment:
Looks like we could use `systems/dnd5e/templates/inventory/encumbrance.hbs` for this, we'd just need to add a options to hide the breakpoints.